### DISCUSSION TITLE
Error: Unknown field name: Comitterdate

### DISCUSSION THREAD

**Discussion by Ashijo** [jesseduffield/lazygit#3367] [Q&A]
I get this error every time I try tout merge or rebase a conflicting branch. Can someone explain what is that error, and maybe how to fix it  ?

**Comment by jakubkaczor** [comment]
Check the correctness of your Git configuration. Specifically, look for the `branch.sort` field. The sorting key is spelled _committerdate_ instead of _comitterdate_ (double “m”).
